ogr/ogrsf_frmts/miramon/mm_gdal_functions.c

```diff
@@ -116,9 +116,9 @@ void MM_FillFieldDescriptorByLanguage(void)
     *(unsigned char *)&szNumberOfVerticesSpa[1] = MM_u_WITH_ACUTE;
     *(unsigned char *)&szNumberOfVerticesSpa[11] = MM_e_WITH_ACUTE;
 
-    CPLStrlcpy(szLenghtOfAarcEng, "Lenght of arc", MM_MAX_IDENTIFIER_SIZE);
-    CPLStrlcpy(szLenghtOfAarcCat, "Longitud de l'arc", MM_MAX_IDENTIFIER_SIZE);
-    CPLStrlcpy(szLenghtOfAarcSpa, "Longitud del arco", MM_MAX_IDENTIFIER_SIZE);
+    CPLStrlcpy(szLengthOfAarcEng, "Length of arc", MM_MAX_IDENTIFIER_SIZE);
```

Comment:
@AbelPau Among all the changes of this commit, this is the only one that isn't just a change in a code comment or variable name, but impacts written data. Is there an issue in interoperability with Miram if fixing "Lenght of arc" to "Length of arc" ? If so, I can revert that change and add an exception for the typo detection script

Reply:
Oh, thanks. In out codes we have length everywhere except this one and other. I'll solve that. You can leave Length with no problem. I have to modifiy some part of the code (slightly) to write the descriptors language independents (I didn't know that but my boss make me note that).